packages/react-renderer/README.md

```diff
@@ -6,11 +6,35 @@ React Renderer Middleware allows for the easy creation of a renderer based on Re
 
 ```txt
 npm i @hono/react-renderer react react-dom hono
-npm i -D @types/react @types/react-dom
+npm i -D @types/react @types/react-dom @vitejs/plugin-react
```

Comment:
I think `@vitejs/plugin-react` is not necessary. Can you remove it and try to run your app again?

Reply:
It's need. Without this plugin, `vscode` will report error. And I checked the official demo from vite, they also have this plugin.

Reply:
Can you try this repo: https://github.com/yusukebe/hono-react-ssr/tree/vite? This may work.

Reply:
I have tried this repo, no error found. Except loss the package dependency `@cloudflare/workers-types`